app/views/users/show.html.erb

```diff
@@ -1 +1,38 @@
-<h1>Here is a profile of a given user</h1>
\ No newline at end of file
+<div class="container">
+    <div class="card mb-3 border-info mb-3" style="max-width: 900px">
+      <div class="row g-0">
+        <div class="col-md-4">
+          <img src=<%@user.name%> class="img-fluid rounded-start">
```

Comment:
- [ ] [OPTIONAL]KIndly ensure you are displaying each user image, this will make your project look good

Reply:
Thank you for a review i'll add the images @Whoistolu